README.md

```diff
@@ -1,106 +1,40 @@
-- **LLM App and Enterprise Features**
-  - [x] App Space: Support hosting Gradio/Streamlit applications and publishing them to App Space.
-  - [x] Fine-grained Permission Control: Fine-grained permission and access control settings for enterprise architecture.
-- **Security Compliance**
-  - [ ] GitServer Adapter: Generic GitServer adapter to support multiple major Git repository types through Adaptor mode.
-  - [x] Asset Metadata: Asset metadata management mechanism, supporting customized metadata types and corresponding AutoTag rules.
-
-The detailed roadmap is designed as follows: [full roadmap](/docs/roadmap_en.md)
-
-### ARCHITECTURE
-CSGHub is made with two typical parts: Portal and Server. This repo corresponds to CSGHub Portal, while CSGHub Server is another high-performance backend project implemented with Golang.
-
-If you want to dive deep into CSGHub Server detail or wish to integrate the Server with your own frontend system or more, you can check the [CSGHub Server open-source project](https://github.com/OpenCSGs/csghub-server).
-
-#### CSGHub Portal Architecture
-<img src="/docs/images/portal_tech_graph.png" width='800'>
-
-#### CSGHub Server Architecture
-<img src="/docs/images/server_tech_graph.png" width='800'>
-
-### QUICK START
-You can refer to [here](/deploy/all_in_one/README.md)quickly deploy a basic csghub instance.
-
-#### Tech docs in detail
-- [setup development env](/docs/setup_en.md)
-
-### Contributing
-We welcome developers of all levels to contribute to our open-source project, CSGHub. If you would like to get involved, please refer to our [contributing guidelines](/docs/CONTRIBUTING_en.md). We look forward to your participation and suggestions.
-
-### ACKNOWLEDGEMENTS
-This project is based on Rails, Vue3, Tailwind CSS, Administrate, Postgresql, Apache Arrow, DuckDB and GoGin, whose open source contributions are deeply appreciated!
-
-### CONTACT WITH US
-If you meet any problem during usage, you can contact with us by any following way:
-1. initiate an issue in github
-2. join our WeChat group by scaning wechat helper qrcode
-3. join our official discord channel: [OpenCSG Discord Channel](https://discord.gg/bXnu4C9BkR)
-4. join our slack workspace:[OpenCSG Slack Channel](https://join.slack.com/t/opencsghq/shared_invite/zt-2fmtem7hs-s_RmMeoOIoF1qzslql2q~A)
-<div style="display:inline-block">
-<img src="/docs/images/wechat-assistant-new.png" width='200'>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-<img src="/docs/images/discord-qrcode.png" width='200'>
-  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-<img src="/docs/images/slack-qrcode.png" width='200'>
-</div>
+
+[![Join WeChat Slack](https://img.shields.io/badge/wechat-join_chat-white.svg?logo=wechat&style=social)](/docs/images/slack-qrcode.png)
```

Comment:
the pic should be `wechat-assistant-new.png`

Reply:
Great catch!